packages/cli/src/utils/command-invocations.js

```diff
@@ -1,47 +0,0 @@
-// this is explicitly left out of utils/index because it creates a circular dependency
-// this file can't be used in any of the required command
-const login = require('../oclif/commands/login');
-const register = require('../commands/register');
-
-const { readCredentials, getLinkedAppConfig } = require('./api');
-const { getInput } = require('./display');
-const _ = require('lodash');
-
-// check if the user needs to be logged in and do it if so
-// returns a promise
-const maybeLogin = context => {
-  return readCredentials(false).then(creds => {
-    if (_.isEmpty(creds)) {
-      context.line('Before you can do that, you need to be logged in.\n');
-      return login.run([]);
-    } else {
-      return Promise.resolve();
-    }
-  });
-};
-
-const hasRegisteredApp = async () => {
-  try {
-    const app = await getLinkedAppConfig();
-    return Boolean(app.id);
-  } catch (err) {
-    return false;
-  }
-};
-
-// check if the user needs to be registered in and do it if so
-// returns a promise
-const maybeRegisterApp = async context => {
-  if (!(await hasRegisteredApp())) {
-    context.line(
-      "Looks like this is your first push. Let's register your app on Zapier."
-    );
-    const title = await getInput('Enter app title (Ctrl-C to cancel):\n\n  ');
-    await register(context, title, { printWhenDone: false });
-  }
-};
```

Comment:
I wonder why is this file removed? Originally we had use cases like calling `maybeRegisterApp` in `zapier push` if the app hasn't been registered. But it looks like we removed those. Maybe there's a reason but I forgot. Can you refresh my memory?

Reply:
good question! During the refactor we pulled that functionality out, so this file wasn't being used. With `registration` being ported, none of the commands this file references exist anymore.

As it is now, commands fail appropriately when auth is wrong or the app isn't registered. So it's not misleading, but not great.

I never loved this previous code approach, so I was going to think through a better way to it in oclif. But for now, we don't need this file anymore.